packages/jest-repl/src/cli/index.ts

```diff
@@ -5,26 +5,27 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
  */
 
 import path from 'path';
 
 import Runtime from 'jest-runtime';
 import yargs from 'yargs';
+// @ts-ignore: Wait for jest-validate to get migrated
 import {validateCLIOptions} from 'jest-validate';
 import {deprecationEntries} from 'jest-config';
 import {version as VERSION} from '../../package.json';
```

Comment:
Oh this as well. The compiler complains that all files should be under the rootDir. Not sure how to resolve this

Reply:
use `require` instead, that should work (I've done it in a couple of places)

Reply:
Yup this works.